components/brave_rewards/resources/rewards_panel/lib/extension_host.ts

```diff
@@ -17,7 +17,8 @@ import { createStateManager } from '../../shared/lib/state_manager'
 import { createLocalStorageScope } from '../../shared/lib/local_storage_scope'
 import * as apiAdapter from './extension_api_adapter'
 
-type LocaleStorageKey = 'catcha-grant-id'
+type LocalStorageKeyCaptchaGrantId = 'catcha-grant-id'
```

Comment:
The "localStorageScope" thing can be used to store more than one keyed value, so this can be:

```ts
type LocalStorageKey = 'captcha-grant-id' | 'load-adaptive-captcha'
```

and then we can use the existing `storage` like:

```ts
storage.readJSON('load-adaptive-captcha', false)
```

Reply:
I should have checked that, thank you - fixed.